applications/crossbar/test/cb_storage_test.erl

```diff
@@ -3,18 +3,70 @@
 -include_lib("eunit/include/eunit.hrl").
 -include("cb_token_restrictions_test.hrl").
 
-%%%=============================================================================
-%%% Generators
-%%%=============================================================================
+-define(ACCOUNT_ID_UNDER_TEST, <<"account0000000000000000000000001">>).
+-define(ACCOUNT_DB_UNDER_TEST, <<"account%2Fac%2Fco%2Funt0000000000000000000000001">>).
+-define(ACCOUNT_DB_CURRENT_MONTH,
+        kazoo_modb:get_modb(kz_util:format_account_id(?ACCOUNT_ID_UNDER_TEST))).
 
-%%------------------------------------------------------------------------------
+%% =======================================================================================
+%% Fixtures / Generators
+%% =======================================================================================
+
+%%--------------------------------------------------------------------
 %% @doc
 %% @end
-%%------------------------------------------------------------------------------
+%%--------------------------------------------------------------------
 maybe_check_storage_settings_test_() ->
+    {setup
+    ,fun setup/0
+    ,fun cleanup/1
+    ,fun (_ReturnOfSetup) ->
+             maybe_check_storage_settings()
+     end
+    }.
+
+%% =======================================================================================
+%% Setup
+%% =======================================================================================
+
+%%--------------------------------------------------------------------
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+setup() ->
+    ?debugMsg(":: Setting up Kazoo FixtureDB"),
```

Comment:
no need for logging, since `kz_fixturedb_util:start_me/0`  is doing this itself.